lib/partition_setup.pm

```diff
@@ -568,9 +576,14 @@ sub take_first_disk_storage_ng {
 
 =head2 take_first_disk
 
- take_first_disk([iscsi => $iscsi]);
+ take_first_disk([%args]);
 
-Selects the first disk in the list to be partitioned
+Take the first disk to be partitioned. Take first storage ng if it is C<is_storage_ng>. 
```

Comment:
Could you rephrase that?
I don't really understand what you want to say in the 2nd sentence.